website/content/en/docs/advanced-topics/custom-bundle-validation.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Custom Bundle Validation
+weight: 80
+---
+
+## Summary
+
+Operator Authors can now use "external" validators with the
+`operator-sdk bundle validate` command by using the
+`--alpha-select-external` flag. This feature enable Operator authors,
+users, and registry pipelines to use custom validators.
+
+## Usage
+
+External validators can be used by specifying a list of local filepaths to
+executables using colons as path separators:
+
+```sh
+operator-sdk bundle validate --alpha-select-external path/validator1:path/validator2
```

Comment:
```suggestion
$ operator-sdk bundle validate --alpha-select-external \
path/validator1:path/validator2
```
Not sure if you want to include the terminal prompt to specify that only user privelages are required. And I am not sure if you break long CLI commands for readability. If so, be sure the other CLI commands for consistency.